src/main/java/org/edumips64/utils/args/EduMipsArgs.java

```diff
@@ -0,0 +1,48 @@
+package org.edumips64.utils.args;
+
+import org.edumips64.Main;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+@Command(mixinStandardHelpOptions = true, resourceBundle = "Messages", versionProvider = EduMipsVersion.class)
+public class EduMipsArgs implements Runnable {
```

Comment:
I have a small naming comment. As far as I remember, we always tried to avoid having `EduMIPS` or similar in the class names, because it's already in the package name. Therefore, could you please rename this class to `Args` and `EduMipsVersion` to `Version`?

Another minor suggestion: I'd suggest to use the `org.edumips64.utils.cli` package rather than `org.edumips64.utils.args`, to convey a bit more the idea that those classes are related to the command-line invocation.

Reply:
Understood, will make those changes for this.

Reply:
Looks good, thanks!